contracts/compliance/modular/modules/MaxBalanceModule.sol

```diff
@@ -117,9 +137,18 @@ contract MaxBalanceModule is AbstractModule {
         address _compliance,
```

Comment:
depending on the amount of Identities on the token we try to preset it could be required to call multiple batches to avoid gas overflow (max gas on EVM blockchains is somewhere between 7 and 25M units, so you cannot put an infinite amount of elements in the array, would be interesting to calculate the average amount of gas and set a limit of array length on the function call to avoid the out of gas error at the start of the function instead of when the function already consumed the max gas amount)